Gopkg.lock

```diff
@@ -53,6 +53,18 @@
   version = "v1.1.0"
 
 [[projects]]
+  branch = "master"
+  name = "github.com/cloudflare/cfssl"
```

Comment:
why do we need gosigar and cfssl?

Reply:
They come with the "license" package.

Reply:
why they are necessary on the client side, what does it do with cffssl?

Reply:
Actually, nothing, that's cause the license package structure is also flat so it drags all deps. Let me see if I can do the same exercise as with reporting and split it.